WebApi/BookService/IBook.cs

```diff
@@ -0,0 +1,50 @@
+﻿// <copyright file="IBook.cs" company="My company">
+//     Copyright (c) My company". All rights reserved.
+// </copyright>
+
+namespace WebApi.BookService
+{
+    using System.Collections.Generic;
+    using WebApi.Models;
+
+    /// <summary>
+    /// CRUD methods for book
+    /// </summary>
+    public interface IBook
+    {
+        /// <summary>
+        /// Get all books
+        /// </summary>
+        /// <returns>List of books</returns>
+        List<Book> GetAllBooks();
```

Comment:
Is interface IBook should know about all books? I think you should use naming like IBookManager or IBookService.